src/Modules/OrchardCore.Commerce/Views/PricePart.Edit.cshtml

```diff
@@ -1,11 +1,11 @@
 @model PricePartViewModel
 
-    <div class="form-group" asp-validation-class-for="Price">
+    <div class="mb-3" asp-validation-class-for="Price">
         <label asp-for="Price">@T["Price"] </label>
         <div class="input-group">
             <input asp-for="PriceValue" class="form-control text-muted" />
             <div class="input-group-append">
-                <select asp-for="PriceCurrency"
+                <select asp-for="PriceCurrency" class="form-select"
                         asp-items="@(new SelectList(Model.Currencies, "CurrencyIsoCode", "CurrencyIsoCode"))"></select>
```

Comment:
Please break each subsequent attribute into a new line if they don't fit in one line.
```suggestion
                <select asp-for="PriceCurrency"
                        class="form-select"
                        asp-items="@(new SelectList(Model.Currencies, "CurrencyIsoCode", "CurrencyIsoCode"))"></select>
```